BDPointXamarinBinder.csproj

```diff
@@ -261,6 +234,7 @@
     <EmbeddedReferenceJar Include="Jars\lifecycle-common-java8-2.2.0.jar" />
     <EmbeddedReferenceJar Include="Jars\converter-moshi-2.7.1.jar" />
     <EmbeddedReferenceJar Include="Jars\retrofit-2.7.1.jar" />
+    <EmbeddedReferenceJar Include="Jars\kotlin-stdlib-1.4.30-RC.jar" />
```

Comment:
Why do we need 1.4.30?

Reply:
Xamarin packages provide Kotlin.Stdlib v1.4.20 and there were few workmanager kotlin missing classes resulting in crash while running the geo trigger.
So added 1.4.30 to fix it